README.md

```diff
@@ -655,38 +746,63 @@
     });
     ```
 
-  - Place 1 space before anonymous function parenthesis.
+  - Place 1 space before the opening parenthesis in control statements (`if`, `while` etc.). Place no space before the argument list in function calls and declarations.
 
     ```javascript
-	// bad
```

Comment:
I really like having a space between the keyword `function` and the `()`. It highlights the fact that you have not named the function and serves as a reminder to do so. The only thing that should be touching the parens is the name of the function itself.

I _could_ be persuaded by an argument that would accept line 763 as the "good" way for function definitions, as one may argue that `name ()` indicates instantiation whereas `name()` indicates execution. But, @tweettypography would have to come here and actually propose that officially ;)